src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class Main {
+
+    public static void main(String[] args) {
+
+        Machine truck = new Truck();
+        Machine bulldozer = new Bulldozer();
+        Machine excavator = new Excavator();
+
+        truck.doWork();
+        bulldozer.doWork();
+        excavator.doWork();
+
+        truck.spotWork();
+        bulldozer.spotWork();
+        excavator.spotWork();
+
```

Comment:
we shouldn't leave the empty line at the end of the method

Reply:
done.